README.md

```diff
@@ -372,6 +372,62 @@ public class WithRegex
 <sup><a href='/src/Destructurama.Attributed.Tests/Snippets.cs#L6-L25' title='Snippet source file'>snippet source</a> | <a href='#snippet-WithRegex' title='Start of snippet'>anchor</a></sup>
 <!-- endSnippet -->
 
+## 8. Working with MetadataTypeAttribute
+
+Apply the attribute `NotLogged` to the property in your MetadataClass. 
+
+This requires to set configuration option UseMetadataTypeAttribute to true.
```

Comment:
```suggestion
You can apply [`MetadataTypeAttribute`](https://learn.microsoft.com/en-us/dotnet/api/system.componentmodel.dataannotations.metadatatypeattribute) to your class providing another type with annotated properties. It may help you in the case when source type with its properties is defined in a code generated file so you don't want to put the attributes in there as they would get overwritten by the generator. Note that you have to set `AttributedDestructuringPolicyOptions.RespectMetadataTypeAttribute` to `true`.
```

Reply:
I tried to dig deeper in functionality and believe current implementation for MetadataTypeAttribute will work only for those Destructuring Attributes which are not accessing the property value.